packages/beacon-state-transition/src/allForks/util/epochContext.ts

```diff
@@ -362,7 +379,36 @@ export class EpochContext {
    */
   exitQueueChurn: number;
 
-  constructor(data: IEpochContextData) {
+  /** TODO: Indexed SyncCommitteeCache */
```

Comment:
this SyncCommitteeCache is same to the old IndexedSyncCommittee, so drop this TODO and below?

Reply:
Fixed